wiki/installing-windows.md

```diff
@@ -10,11 +10,29 @@ Once the boot menu is open, move the cursor using the arrow keys to the name of
 
 ¹ Can be usually USB HDD or your USB drive model. Your mileage may vary.
 
-## Installing
+## Disk Conversion (if needed)
 
-::: danger
-The disk that you are trying to install Windows on must be a GPT disk if you are on a UEFI system or MBR if you are on a BIOS system, otherwise it will not let you install. Use `diskpart` to change the partition scheme. ***THIS DELETES ALL DATA ON THE DISK***
-:::
+The disk that you are trying to install Windows on must be a GPT disk if you are on a UEFI system or MBR if you are on a BIOS system, otherwise it will not let you install. 
+
+Use DiskPart to change the partition scheme. ***THIS MAY DELETE ALL DATA ON THE DISK. MAKE SURE YOU HAVE BACKED UP YOUR DATA BEFORE PROCEEDING***
+
```

Comment:
```suggestion
Use DiskPart to change the partition scheme.

::: danger CAUTION
This may **DELETE ALL DATA** on the disk! Make sure you have a backup before you proceed.
```